packages/react/error-boundary/src/hooks/useIsChanged.ts

```diff
@@ -0,0 +1,4 @@
+/** @tossdocs-ignore */
+import { usePrevious } from '@toss/react';
+
+export const useIsChanged = (value: unknown) => usePrevious(value) !== value;
```

Comment:
I made custom hook, `useIsChanged` to check whether value is changed with previous rendering value.

this hook will check parent ErrorBoundaryGroup.resetKey is changed
If parentErrorBoundaryGroup.resetKey is changed, ErrorBoundaryGroup self will change resetkey to propagate to child ErrorBoundaryGroup recursively.